### ISSUE TITLE
code actions: add alias for undefined function

### ISSUE THREAD

**Issue by mhanberg** [elixir-tools/next-ls#425]
## Description

TODO: just getting this up here for now

cc @srcrip

**Comment by mhanberg** [comment]
@srcrip does this sound like the problem you were describing and if yes, a desirable solution?

**Comment by srcrip** [comment]
Yes this is perfect, this is the general gist of what I was looking for.

Might I suggest exposing a configuration option to search only under the main module of the current application too? Or instance if you're currently in Foo.Bar search only Foo and its children? I think that could potentially speed up a lot in big projects. But you'd definitely want to be able to configure that.

**Comment by mhanberg** [comment]
can definitely scope it to just your application code, rather than all modules. probably better, as we can query for them in the database.
